wp/plugin/lib/wp_cache.ml

```diff
@@ -32,87 +30,14 @@ module Digests = struct
 
   (* Creates a digest for the knowledge cache. *)
   let knowledge (mk_digest : namespace:string -> digest) : digest =
-    mk_digest ~namespace:"knowledge"
+    mk_digest ~namespace:"wp-knowledge"
```

Comment:
Were there namespace clashes here?

Reply:
no, actually, It was a hail Mary to try to understand the bug I'm getting.